examples/generalized_linear_models/GLM-linear.ipynb

```diff
@@ -103,9 +103,6 @@
    "metadata": {},
```

Comment:
<p>Even though we only use it once, I would keep the rng creation in the 2nd cell plus consequent usage of the <code>rng</code>  object, like the example in https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Sure, I'll add that.